src/main/main.ts

```diff
@@ -0,0 +1,70 @@
+import { PROJECT_ID } from "./basecamp";
+import { generateIdForRow, getBasecampTodoRequestsForRow, getId, hasId, saveRow } from "./row";
+import { getEventRowsFromSpreadsheet } from "./scan";
+import { createTodo, TODOLIST_ID } from "./todos";
+
+const DEFAULT_TODOLIST_IDENTIFIER: TodolistIdentifier = {
+    projectId: PROJECT_ID,
+    todolistId: TODOLIST_ID
+};
+
+/**
+ * Main entry point for the Onestop to Basecamp Integration that contains the core logic for
+ * adding/updating/deleting Basecamp Todos based on the rows on the Onestop
+ */
+export function main(): void {
```

Comment:
later we can think of a better name for this - this function will be called both manually from a menu and periodically based on a schedule. importOnestopToBascamp?

Reply:
i'm actually ok with main too

Reply:
Didn't see the followup comment. Renamed to `importOnestopToBascamp `